code/game/objects/items/control_wand.dm

```diff
@@ -32,12 +32,16 @@
 	switch(mode)
 		if(WAND_OPEN)
 			mode = WAND_BOLT
+			desc = "Remotely controls airlocks. Current mode: toggle bolts"
```

Comment:
You should do all this in the `examine` proc instead of setting `desc` manually. An example would be:
```dm
/obj/item/door_remote/examine(mob/user)
	. = ..()
	. += "<span class='notice>It's current mode is: [mode]</span>
```

Reply:
alright, i will do that.